atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -342,27 +344,62 @@ private static SerializableTransactionManager create(
         return transactionManager;
     }
 
-    private static SweepBatchConfig getSweepBatchConfig(AtlasDbRuntimeConfig config) {
+    private static void validateRuntimeConfig(@Nullable AtlasDbRuntimeConfig runtimeConfig) {
+        if (runtimeConfig == null) {
+            log.warn("This service uses live reload configs. Please use the 'atlas-runtime' block for specifying"
+                    + " live-reloadable atlas configs. Default to using configs specified on the 'atlas' block");
+            return;
+        }
+
+        if (runtimeConfig.enableSweep() == null) {
+            log.warn("Use the 'enableSweep' config on the atlas-runtime block to make it live-reloadable");
```

Comment:
In my opinion, these should only be flagged if the corresponding non-runtime config is set. In practice, we aim for most customers being fine with the defaults, meaning that these warnings would create a lot of noise.